### PR TITLE
added home page

### PR DESCRIPTION
**Pull request by hiteshverma27**

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -1,51 +1,5 @@
...
-.links a {
-  color: #d9d9d9;
-  font-size: 1.2rem;
-  margin: 1rem;
-}
+@import url("https://lightsout-store.netlify.app/styles/login.css");
+@import url("https://lightsout-store.netlify.app/style.css");
+@import url("https://lightsout-store.netlify.app/styles/product.css");
+@import url("https://lightsout-store.netlify.app/styles/wishlist.css");
+@import url("https://lightsout-store.netlify.app/styles/cart-management.css");
```
Comment (Aditi0505) [inline src/App.css:5]: Please change this as suggested by Kishan.

File: src/components/Footer/Footer.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+        </div>
+        <div className="mb-auto my-5">
+          <h2>Account</h2>
+          <hr className="my-2 hr-color-accent" />
+          <ul className="font-size-large font-color-grey">
+            <li className="my-1"><a href="https://github.com/">My Account</a></li>
+            <li className="my-1"><a href="https://github.com/">Wishlist</a></li>
+            <li className="my-1"><a href="https://github.com/">Track order</a></li>
+            <li className="my-1"><a href="https://github.com/">View Cart</a></li>
+            <li className="my-1"><a href="https://github.com/">Priacy Policy</a></li>
```
Comment (Aditi0505) [inline src/components/Footer/Footer.js:43]: There is a lot of repetition over here. You can create a component for this, if you have bandwidth.

File: src/components/Navigation/Navigation.js [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+function Navigation() {
+    return <nav className="w-100vw flex-space_between-center px-10 py-1 bg-white" id="nav">
+        <a href="/index.html">
+            <h2>LightsOut</h2>
+        </a>
+        <input type="search" className="input w-40per" placeholder="Search LightsOut" id="search-bar-nav" />
+        <ul className="flex-space_between-center" id="nav-icons">
+            <li className="mx-2">
+                <a href="/Pages/login.html"><button className="btn-primary-confirm">Login</button></a>
```
Comment (Aditi0505) [inline src/components/Navigation/Navigation.js:11]: Going forward you can use Link instead.

**Review by Aditi0505 — APPROVED** [review]
Left few suggestion for you to look at. Rest looks good to me.
Also, please add proper description and screenshots for better visualization.

File: src/App.js [hunk 4]
```diff
@@ -1,42 +1,15 @@
 import "./App.css";
-import logo from "./logo.png";
+import { Footer } from "./components/Footer/Footer";
+import { Navigation } from "./components/Navigation/Navigation";
+import { Home } from "./pages/Home/Home";
```
Comment (rohanmathur91) [inline src/App.js:4]: try barrel exports.

File: src/components/Footer/Footer.js [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+          </ul>
+        </div>
+        <div className="mb-auto my-5 pr-3">
+          <h2>Contact</h2>
+          <hr className="my-2 hr-color-accent" />
+          <ul className="font-size-large font-color-grey">
+            <li className="my-1"><strong>Location:</strong> India </li>
+            <li className="my-1"><strong>Call:</strong> +91-9090909090 </li>
+            <li className="my-1"><strong>Email:</strong> support @lightsout.com </li>
+          </ul>
```
Comment (rohanmathur91) [inline src/components/Footer/Footer.js:53]: Have a array of data and just loop over it

File: src/components/HomeMain/HomeMain.js [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+      description:
+        "Non-fiction is writing that gives information or describes real events, rather than telling a story.",
+    },
+    {
+      _id: uuid(),
+      categoryName: "horror",
+      description:
+        "Meant to cause discomfort and fear for both the character and readers, horror writers often make use of supernatural and paranormal elements in morbid stories that are sometimes a little too realistic.",
+    },
+  ];
```
Comment (rohanmathur91) [inline src/components/HomeMain/HomeMain.js:24]: Put this in separate file and import and use it

File: src/pages/Home/Home.js [hunk 7]
```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+import { Header } from '../../components/Header/Header'
+import { HomeMain } from '../../components/HomeMain/HomeMain'
```
Comment (rohanmathur91) [inline src/pages/Home/Home.js:3]: try barrel exports

**Review by rohanmathur91 — APPROVED** [review]
Added few suggestions

File: src/components/Navigation/Navigation.js (hunk 3 above)
Comment (hiteshverma27) [reply src/components/Navigation/Navigation.js:11]: I will rectify the code, thanks for pointing out.

**Pull request merged**